osaca/semantics/hw_model.py

```diff
@@ -312,6 +454,7 @@ def get_isa_for_arch(arch):
     def dump(self, stream=None):
```

Comment:
Why is this whole function commented?

Reply:
Still needs work, marked unresolved.

Reply:
5f9de2c41d8c2eb91564dd44574f114b735a83f3